ods_ci/tests/Resources/Page/DistributedWorkloads/WorkloadMetricsUI.resource

```diff
@@ -333,6 +333,23 @@ Check Distributed Worklaod Status Overview
     Check Expected String Equals    //td[@data-label="Status"]//span[@class="pf-v5-c-label__text"]    ${job_status}
     Check Expected String Equals    //td[@data-label="Latest Message"]    ${job_status_message}
 
+Check Distributed Worklaod Status Upgrade Overview
+    [Documentation]    checks Distributed Worklaod Status Overview displaying correctly
+    [Arguments]    ${job_name}    ${job_status}    ${job_status_message}
+    Click Button    ${WORKLOAD_STATUS_TAB_XP}
+    Wait Until Element Is Visible    xpath=//div[text()="Distributed Workloads"]    timeout=20
+    Check Expected String Equals    //*[@id="status-overview-ChartLabel-title"]    1Distributed Workloads
+    Page Should Contain    text=${JOB_STATUS}: 1
+    Check Expected String Contains    //td[@data-label="Name"]    ${job_name}
+    Check Expected String Equals    //td[@data-label="Priority"]    0
+    ${current_date} =    Get Current Date    result_format=%m/%d/%Y
+    ${ui_fetched_datetime} =    Get Text  xpath=//td[@data-label="Created"]
```

Comment:
Create a separate method for L346-L349 as like we have https://github.com/red-hat-data-services/ods-ci/blob/8b4077e8d5e10591546ead692763e1ce29d9a4f7/ods_ci/tests/Resources/Page/DistributedWorkloads/WorkloadMetricsUI.resource#L332

Reply:
Hey @ChughShilpa , Thanks for your review!
Yes, you're right! The repetition is not needed in this case so I 'm nowe utilising previous keyword with some tweaks now it should work.
I have created separate keyword for above mentioned lines.. Please review!